review/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# This file is based on https://github.com/kmuto/review/blob/master/Dockerfile.
+# See https://github.com/kmuto/review/blob/master/COPYING for License.
+
+FROM debian:sid
```

Comment:
Do we have a way to base it on alpine and use multi stage so we reduce image size?

Reply:
official Dockerfile is based on debian:sid, and installs some dependencies packages such as git. I can change the base image to alpine, but I can not ensure that it will be executed correctly.